batchlib/workflows/cell_analysis.py

```diff
@@ -284,6 +295,14 @@ def cell_analysis_parser(config_folder, default_config_name):
     parser.add("--ignore_invalid_inputs", default=None)
     parser.add("--ignore_failed_outputs", default=None)
 
+    # MongoDB client config
+    parser.add("--db_username", type=str, default='covid19')
+    parser.add("--db_password", type=str, default='')
```

Comment:
Is this a good idea? Looks like it might encourage people to write pws into the configs directly....
Is there someway to use a `~/.mongorc` or similar?

Reply:
yeah, I though about it. The only thing we could do is pass `DB_PASSWORD` as env var. But I'd leave it like this for convenience now and trust that the password is always passed as a command line arg.

Reply:
Ok, find with me for now. If this ever becomes more security critical, we can reconsider.